yocto-update-strategies.md

```diff
@@ -0,0 +1,651 @@
+project lifecycle we discussed earlier. During the initiation phase,
+architecture documentation is created, providing a foundational understanding of
+the system structure. This documentation becomes particularly valuable during
+updates, helping us identify areas that may require modifications or
+improvements.
+
+Moving into the planning phase, features documentation is developed, outlining
+the desired functionalities of the project. This documentation guides us in
+prioritizing updates based on project requirements and ensures that our updates
+align with the intended goals of the system.
+
+As we progress into the development phase, the build environment is established.
+It provides the necessary tools, configurations, and dependencies for building
+and updating the project. Maintaining an up-to-date build environment is crucial
+for efficient updates, ensuring compatibility with the latest Yocto releases.
+
+During the testing phase, comprehensive tests are designed and implemented.
+These tests validate the project's functionality and help us identify potential
+regressions that may arise during updates. By conducting thorough tests, we can
+ensure that the system remains stable and reliable after applying updates.
+
+In the deployment phase, CI/CD processes come into play. These automated
+workflows streamline the build and testing stages, making the deployment of
+updates more efficient and reproducible. CI/CD ensures that updates are
+integrated seamlessly into the system.
+
+Finally, all of these project components, including architecture documentation,
+features documentation, build environment, tests, and CI/CD, collectively
+contribute to ongoing maintenance and updates. They form a cohesive framework
+that supports the project throughout its lifecycle, enabling us to address
+security vulnerabilities, minimize technical debt, and integrate new
+functionalities as we maintain and update our Yocto-based projects.
+
+---
+# Update strategy
+
+* Remembering every project component during updates
+  - each project component, such as architecture documentation, features
+    documentation, build environment, tests, and CI/CD, plays a crucial role in
+    the project's functionality and stability
+  - updates must consider all components to ensure compatibility, minimize
+    regressions, and maintain the overall integrity of the Yocto-based project
+* Importance of the update deployment method
+  - the update deployment method impacts the efficiency, scalability, and user
+    experience of the Yocto-based project
+  - having a dedicated update system allows for streamlined and controlled
+    updates, minimizing downtime and reducing the need for manual intervention
+  - evaluating the requirements and constraints of the project helps determine
+    whether a dedicated update system is necessary or if reflashing devices is
+    feasible
+
+???
+
+TBD notes
+
+---
+# Update strategy
+
+* Yocto release cycle
```

Comment:
Same as above
Wall of text on the slide is not the best way to go

Reply:
also fixed https://github.com/3mdeb/conferences/pull/7/commits/47359d083f2e6c8919c2292127ced92fe98d1571